ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -796,6 +796,11 @@ def less_equal(self, y, name=None):
     def real(self, name=None):
         return paddle_frontend.real(self._ivy_array)
 
+    @with_unsupported_dtypes({"2.5.1 and below": ("float16", "bfloat16")}, "paddle")
+    def t(self, name=None):
+        axes = list(range(len(self._ivy_array.shape)))[::-1]
```

Comment:
```suggestion
        axes = list(range(len(self.ivy_array.shape)))[::-1]
```
We shouldn't access a private attribute :)